securing_apps/topics/oidc/java/jboss-adapter.adoc

```diff
@@ -116,10 +120,14 @@ $ ./bin/jboss-cli.sh --file=adapter-elytron-install-offline.cli
 ----
 $ ./bin/jboss-cli.sh --file=adapter-install-offline.cli
 ----
+
+NOTE: It is possible to use legacy non-elytron adapter on Wildfly 11 or newer as well, hence you can use `adapter-install-offline.cli`
+even on those versions. However we recommend to use newer Elytron adapter.
```

Comment:
I have a couple minor changes to suggest here:

NOTE: It is possible to use the legacy non-Elytron adapter on Wildfly 11 or newer as well, meaning you can use `adapter-install-offline.cli` even on those versions. However, we recommend to use the newer Elytron adapter.